format_ini.py

```diff
@@ -33,7 +33,7 @@ def _format_file(filename: str) -> int:
     with open(filename, encoding="UTF-8", newline="") as f:
         contents = f.read()
 
-    orig = configparser.RawConfigParser()
+    orig = configparser.RawConfigParser(strict=False)
```

Comment:
this looks a little spooky but all it seems to control is duplicates: https://docs.python.org/3/library/configparser.html#configparser.ConfigParser